optimum/exporters/openvino/convert.py

```diff
@@ -662,7 +657,38 @@ def export_from_model(
                     "please install it with `pip install nncf`"
                 )
 
-    if library_name != "diffusers":
+    if library_name == "open_clip":
+        models_and_export_configs = {}
+
+        visual_model = model.visual
+        setattr(visual_model, "config", model.config.vision_config)
+        vision_cfg = OpenCLIPVisualOpenVINOConfig(
+            model.config.vision_config, "zero-shot-image-classification", preprocessors
+        )
+        models_and_export_configs["vision"] = (visual_model, vision_cfg)
+
+        text_model = model.text
+        setattr(text_model, "config", model.config.text_config)
+        text_cfg = OpenCLIPTextOpenVINOConfig(
+            model.config.text_config, "zero-shot-image-classification", preprocessors
+        )
+        models_and_export_configs["text"] = (text_model, text_cfg)
+
```

Comment:
could this be moved in a separate function as done for other models  with `_get_submodels_and_export_configs`

Reply:
moved